tests/testthat/test-util.R

```diff
@@ -41,10 +41,10 @@ test_that("sync_and_drop_caches() works", {
     }
   }
 
-  cases = purrr::cross(list(
+  cases = suppressWarnings(purrr::cross(list(
```

Comment:
`purrr::cross()` is depressingly [getting deprecated](https://github.com/tidyverse/purrr/issues/768). I looked at alternatives, but they involve adding deps and going through dataframes for what should be simple, so I'll probably rewrite a version from scratch whenever this starts actually failing. Till then, this keeps the tests passing.